LIC_FILES_CHKSUM.sha256

```diff
@@ -4,6 +4,7 @@ bbb303820971c294a9a8e5eba5affcf1379036e877ea61c11cbf9400b2949483  vendor/github.
 bbb303820971c294a9a8e5eba5affcf1379036e877ea61c11cbf9400b2949483  vendor/github.com/mendersoftware/log/COPYING
 3591f687e2d6f49c83b1ec69577e8110afbde80be5ec81791bd86d2838ccd3de  vendor/github.com/mendersoftware/scopestack/LICENSE
 bbb303820971c294a9a8e5eba5affcf1379036e877ea61c11cbf9400b2949483  vendor/github.com/mendersoftware/scopestack/COPYING
+192def682f6846c6a57e20cf044e224fa467948358163a18dd37bfe14578e6ab  vendor/github.com/pkg/errors/LICENSE
```

Comment:
You can put this under a headline `# These are identical to BSD 2 Clause license.`, since that's what it is.